src/main/java/pl/byczazagroda/trackexpensesappbackend/service/WalletDTOMapper.java

```diff
@@ -0,0 +1,18 @@
+package pl.byczazagroda.trackexpensesappbackend.service;
+
+import org.springframework.stereotype.Service;
+import pl.byczazagroda.trackexpensesappbackend.dto.WalletDTO;
+import pl.byczazagroda.trackexpensesappbackend.model.Wallet;
+
+import java.time.Instant;
+
+@Service
+class WalletDTOMapper {
```

Comment:
In this project we are going to use MapStruct. So you don't have to implement it yourself.

Reply:
Custom WalletDTOMapper class has been replaced with MapStruct